ambry-api/src/main/java/com/github/ambry/quota/AmbryEnforcementRecommendation.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2021 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.quota;
+
+/**
+ * An implementation of {@link EnforcementRecommendation} made by Ambry.
+ */
+public class AmbryEnforcementRecommendation implements EnforcementRecommendation {
```

Comment:
I still think it's not necessary to have EnforcementRecommendation interface. This recommendation is not a class of "how to do" but "what info to pass". I can't think of any different implementation rather than AmbryEnforcementRecommendataion.

Reply:
removed interface.